src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,10 +8,11 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street, #12-34, 654321";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in the format block"
+    public static final String EXAMPLE = Block.EXAMPLE + ", " + Street.EXAMPLE + ", " + Unit.EXAMPLE + ", "
+                        + PostalCode.EXAMPLE;
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in the format: block"
                                 + ", street, unit, postal code. Each seperated by a comma.";
-    public static final String ADDRESS_VALIDATION_REGEX = "[\\w]+,[\\s\\w]+,[\\s\\d#-]+,[\\s\\w]+";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+, .+, .+, .+"; //"[\\w]+,[\\s\\w]+,[\\s\\d#-]+,[\\s\\w]+";
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.